sorc/chgres_cube.fd/program_setup.f90

```diff
@@ -443,6 +445,25 @@ end subroutine get_var_cond
 !> Driver routine to compute soil parameters for each
 !! soil type. Works for Zobler and STATSGO soil categories.
 !!
+!! The calculations are those used in the Noah Land Surface Model. For
```

Comment:
I can build your branch with no warnings.  But the hyperlinks are not displaying right.  Some documentation I found said the link should be enclosed in ( ) and the wording in the square brackets.

Reply:
Stand by, I got that backwards. I will fix...

Reply:
OK, the href tag from html seems to work best. I have changed to that.